lib/reline/io/dumb.rb

```diff
@@ -0,0 +1,105 @@
+require 'io/wait'
+
+class Reline::Dumb < Reline::IO
+  RESET_COLOR = '' # Do not send color reset sequence
+
+  def initialize(encoding: Encoding.default_external)
```

Comment:
I think default value of encoding should be nil, or we need to check RUBY_PLATFORM.
If `@encoding` is always set, we can't get into `elsif RUBY_PLATFORM =~ /mswin|mingw/` branch in `def encoding` and this seems to be the reason CI (irb) is failing. (but I can't reproduce the failing in my local environment...)